g11n-ws/modules/md-data-api-s3impl/build.gradle

```diff
@@ -35,7 +35,8 @@ repositories {
 
 dependencies {
     compile project(":md-data-api")
-	compile project(":vip-common")
+    compile project(":vip-common")
+    compile project(":md-data-s3util")
```

Comment:
why change the i18n codes

Reply:
Here added a new project 'md-data-s3util' for common use among I18N and L10N.